packages/front-end/pages/experiments/index.tsx

```diff
@@ -67,6 +67,7 @@ const ExperimentsPage = (): React.ReactElement => {
     ],
     transforms
   );
+  const experimentsHaveProjects = experiments.some((e) => e.project);
```

Comment:
In addition to this, I think we only want to show the column when "All Projects" is selected in the left nav.  If you have a specific project selected, the column is not very useful since it will be the same for every row. 

I would change the variable name too:

```ts
// If "All Projects" is selected and some experiments are in a project, show the project column
const showProjectColumn = !project && experiments.some((e) => e.project);
```

This same thing goes for features.